R/reportEmi.R

```diff
@@ -143,8 +143,11 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL,
   # FE non-energy use
   vm_demFENonEnergySector <- readGDX(gdx, "vm_demFENonEnergySector", field = "l",
                                      restore_zeros = F, react = "silent")[,t,]
+
   if (length(vm_demFENonEnergySector) == 0) {
     vm_demFENonEnergySector <- NULL
+  } else {
+    vm_demFENonEnergySector <- expandMagclass(vm_demFENonEnergySector, vm_demFeSector)
```

Comment:
as this helper also removes values from third dimension that are not in `vm_demFeSector`, i am not sure if the trimming should be done here or only further down below in the context of the condition where it crashes (line 304)

Reply:
Here is good.